README.md

```diff
@@ -23,27 +23,39 @@ speed!
   + [Testing with Multiple Python Versions](#testing-with-multiple-python-versions)
   + [Code Linting](#code-linting)
 * [Running GatorGrader](#running-gatorgrader)
+* [Using Docker](#using-docker)
 * [Comparison to Other Tools](#comparison-to-other-tools)
 * [Presentations](#presentations)
 * [Contributing](#contributing)
 * [Contributors](#contributors)
 
 ## Quickstart Guide
-
-Want to quickly get up and running with GatorGrader? If you have an existing
-assignment and you want to automatically check its source code or technical
-writing, then you can use GatorGrader's [Gradle
-plugin](https://github.com/GatorEducator/gatorgradle). The following starter
-repositories provide examples of how to configure GatorGrader to check programs
-and documentation for different languages:
-[Java](https://github.com/GatorEducator/java-starter),
-[LaTeX](https://github.com/GatorEducator/latex-assignment-starter), and [HTML
-with CSS](https://github.com/GatorEducator/html-css-assignment-starter). These
+* Starter Repositories
+An easy way to get started with GatorGrader is to check out our sample starter repositories.
+The following starter repositories provide examples of how to configure GatorGrader
```

Comment:
the language "how to configure GG to do [blank]" is a bit clunky
Could this be re-phrased so the word "to" is not repeated?

*same in the first sentence of solution repos